### ISSUE TITLE
Missing `Shell Integration Decoration` in windows 11

### ISSUE THREAD

**Issue by Lakshmikanth2001** [microsoft/vscode#154674]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: `1.69.0`
- OS Version: `windows 11`

![2022-07-10](https://user-images.githubusercontent.com/52835045/178137025-323bb8e3-587f-4294-bb55-97eea9b6e0a1.png)

Steps to Reproduce:

1. Enable `Shell Integration` in `vscode` on `windows`
2. launch `PowerShell` and enter some terminal commands

**Comment by gjsjohnmurray** [comment]
Maybe follow the on-screen advice about updating your Power Shell.

**Comment by Lakshmikanth2001** [comment]
## Thank you sir after updating `PowerShell` shell integration started working 
- Updated `PowerShell` to enable  vscode `Terminal: shell Integration Decorators`

![shellIntegrationWorking](https://user-images.githubusercontent.com/52835045/178137957-dafa42ce-2c14-4797-9e65-dad5747ff940.png)

**Comment by Tyriar** [comment]
Yep this was because you were on "Windows PowerShell", not "PowerShell". I've also found the naming pretty confusing as "Windows PowerShell" used to be called "PowerShell", when the now "PowerShell" was "PowerShell Core" 🤯

**Comment by Tyriar** [comment]
Closing again as there's nothing to do here, it's not possible unfortunately to support windows powershell afaict
